src/Callbacks.php

```diff
@@ -6,6 +6,16 @@
 
 class Callbacks
 {
+    /**
+     * Return a HTTP 401 Unauthorized response.
+     *
+     * @throws \Symfony\Component\HttpKernel\Exception\HttpException
+     */
+    public static function unauthorized()
+    {
+        throw new HttpException(401);
+    }
```

Comment:
You could use `Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException` for `unauthorized` and `Symfony\Component\HttpKernel\Exception\NotFoundHttpException` for `notFound`. This is not mandatory but can make the exception more talkative than `401`.

Reply:
This is a good point, i will submit another pull request with the change.